platform-cloud/docs/studios/overview.md

```diff
@@ -91,19 +91,48 @@ To install additional Python packages during a running Studio session, execute `
 
 To see the list of all JupyterLab image templates available, including security scan results or to inspect the container specification, see [public.cr.seqera.io/repo/platform/data-studio-jupyter][ds-jupyter].
 
-**RStudio Server 4.4.1**
+### RStudio Server 4.4.1
 
 The default user is the `root` account. To install R packages during a running Studio session, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
 
 To see the list of all RStudio Server image templates available, including security scan results or to inspect the container specification, see [public.cr.seqera.io/repo/platform/data-studio-rstudio][ds-rstudio].
 
-**Visual Studio Code 1.93.1**
+### Visual Studio Code 1.93.1
 
 [Visual Studio Code][def-vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. The container template image ships with the latest stable version of [Nextflow] and the [VS Code extension for Nextflow][nf-lang-server] to make troubleshooting Nextflow workflows easier. To install additional extensions during a running Studio session, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
 
 To see the list of all Visual Studio Code image templates available, including security scan results or to inspect the container specification, see [public.cr.seqera.io/platform/data-studio-vscode][ds-vscode].
 
-**Xpra 6.2.0**
+#### Docker-in-docker
+
+A primary use case for VS Code in Studios is to develop new, and troubleshoot existing, Nextflow pipelines. This commonly requires running Docker in the Dockerized container. The recommended method is to:
```

Comment:
1. and 2. aren't consecutive steps - are they linked or are they two separate suggestions?

Reply:
The leading sentence of "The recommended method is to:" matches with step 1, as it introduces an instruction which starts with a verb ("Create"). 

Step 2 should then start with a verb; otherwise, if we removed step 1, it would read as "The recommended method is to: One running Studio...".

Reply:
The two items don't seem linked unless you do one first and then the other, with step 2 depending on step 1 being performed first.

Reply:
All that said, I've made a change that I believe addresses my concern. LMKWYT.